.github/workflows/Test.yml

```diff
@@ -313,13 +313,75 @@ jobs:
     - name: '🚧 Test package'
       run: GHDL=ghdl ./testsuite/testsuite.sh ${{ matrix.suite }}
 
+#
+# pyGHDL DOM Test
+#
+
+  DOM-test:
+    needs: win-build
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 8
+      matrix:
+        sys: [
+          {icon: '🟦', installs: 'MINGW32', arch: i686,   pkg: 'mcode' },
+          #{icon: '🟦', installs: "MINGW32", arch: i686,   pkg: "llvm"  }, ! Not yet functional
+          #{icon: '🟪', installs: "MINGW64", arch: x86_64, pkg: "mcode" }, ! mcode is not yet supported on win64
+          {icon: '🟪', installs: 'MINGW64', arch: x86_64, pkg: 'llvm'  },
+        ]
+    name: '${{ matrix.sys.icon }} DOM · ${{ matrix.sys.installs }} · ${{ matrix.sys.pkg }} · ${{ matrix.suite }}'
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+
+    - name: '${{ matrix.sys.icon }} Setup MSYS2'
+      uses: msys2/setup-msys2@v2
+      with:
+        msystem: ${{ matrix.sys.installs }}
+        update: true
+        install: >
+          mingw-w64-${{ matrix.sys.arch }}-python-pip
+          mingw-w64-${{ matrix.sys.arch }}-python-setuptools
+
+    - name: '⚙️ git config'
+      run: git config --global core.autocrlf input
+      shell: bash
+
+    - name: '🧰 Checkout'
+      uses: actions/checkout@v2
+
+    - name: '📥 Download artifact: package'
+      uses: actions/download-artifact@v2
+
+    - name: '🚧 Install package and Python dependencies'
+      run: |
+        pacman --noconfirm -U artifact/mingw-w64-${{ matrix.sys.arch }}-ghdl-${{ matrix.sys.pkg }}-*.zst
+        pip3 install .
+
+    - name: Test the DOM and pretty print all VHDL sources
+      if: matrix.suite == 'sanity pyunit vpi vhpi'
```

Comment:
Unlike `win-test` (where this was presumably copied from), this job doesn't define `matrix.suite`, so this is never run.

Reply:
Interesting, you are correct, but the jobs did run indeed 😆 (see below). It's obviously a bug in GitHub's evaluation of the `if`.

I will fix that.

Reply:
It's just the step that is skipped: https://github.com/ghdl/ghdl/pull/1797/checks?check_run_id=2857078893

Reply:
Correct. My refactoring mistake: https://github.com/umarcor/ghdl/runs/2854455105?check_suite_focus=true#step:8:18.

Running now: https://github.com/ghdl/ghdl/runs/2857276874?check_suite_focus=true#step:7:18